### DISCUSSION TITLE
Feat: Icon Packages

### DISCUSSION THREAD

**Discussion by suitupstan** [EddieHubCommunity/LinkFree#417] [Q&A] [answered]
This is a poll! The one with the most wits wins the match

- 👍: Font Awesome - https://fontawesome.com/
- 👎: Devicons - https://devicon.dev/
- 😄: Simple Icons - https://simpleicons.org/
- 🎉: Feather Icons - https://feathericons.com/
- 😕: css.gg - https://css.gg/
- ❤️: Material Design Icons - https://materialdesignicons.com/
- 🚀: HeroIcons - https://heroicons.com/
- 👀: Icomoon - https://icomoon.io/
- Upvote: Bootstrap Icons - https://icons.getbootstrap.com/

**Comment by suitupstan** [comment]
React to add votes: 
![image](https://user-images.githubusercontent.com/91655303/135705792-cddf9df9-532b-4399-b550-fe2488762ebc.png)

**Comment by suitupstan** [comment]
React icons is not included (https://react-icons.github.io/react-icons/) since it feels more like a combined library for all of them

**Comment by suitupstan** [comment]
I think there has been enough time since this started... ending the voting, the winner is: "Font Awesome"

**Reply by eddiejaoude** [reply]
I still think React icons has more options, but we can always change it later if we need to

**Reply by suitupstan** [reply]
Hey @eddiejaoude React Icons is more of a aggregator for the icons.

**Reply by Vyvy-vi** [reply]
At the same using that aggregator has a lot of benefits than sticking to one icon set and including it's specific support module for react in the project...

**Reply by suitupstan** [reply]
> At the same using that aggregator has a lot of benefits than sticking to one icon set and including it's specific support module for react in the project...

We could use React-Icons but we still need to choose an icon set 😅 Or we could make it like '[ICON_PACKAGE]-[ICON_NAME]', example: `fa-book`, `pi-book`

**Reply by Vyvy-vi** [reply]
I mean, can't we integrate any of them along with react-icons...

It isn't hard to switch between icon sets that are included in react-icons...
And if needed, we can even use multiple sets, if a particular set lacks an icon.

Imo, since FontAwesome seems to be the popular choice, we should probably set that up with React-icons at the earliest...

**Comment by Panquesito7** [comment]
Devicon is actually a pretty good iconset project, I think we should use that.
In case you need any new icons, please contact me! I'm a maintainer there. 🙂

**Comment by eddiejaoude** [comment, marked as answer]
I think React icons where picked https://github.com/EddieHubCommunity/LinkFree/blob/main/package.json#L15
